transformations/urban_dict_swap/test.json

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "urban_dict_swap",
+    "test_cases": [
```

Comment:
It is recommended to include at least 5 examples here.

Reply:
Hi, thanks for the review. I'm afraid I can't add anything to the test.json, since the build pipeline interprets them as static test cases. Since the transformation is non-deterministic, it'll fail the build.

On the point about tagging, I would argue that swapping out nouns for synonyms is quite a useful perturbation when training a POS tagger.

Since this wholly relies on an API which I did not develop, I am not comfortable claiming ownership over the whole transform. However, I have identified myself as the author in the README.

Reply:
I've added a text generation benchmark to the readme in 2de3222

Reply:
Cool, thanks for the changes.

Reply:
Okay, if the build is failing, you can add sample examples in the README. It will give an idea of what this transformation generates - both good and bad.